coalib/bears/Bear.py

```diff
@@ -47,6 +47,13 @@ class Bear(Printer, LogPrinter):
     >>> class SomeBear(Bear):
     ...     LANGUAGES = "Java"
 
+    To see what requirements the bear has, give the bear a value for
+    ``REQUIREMENTS``, which will be a tuple of strings:
+
+    >>> class SomeBear(Bear):
+    ...     REQUIREMENTS = ('yamllint~=1.0.1')
```

Comment:
this would make a string with a comma. So either you want a plain string (--> no braces at all) or a tuple (braces + comma)